.github/workflows/main.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         run: |
           source ./.venv/bin/activate
           python -m connector_packager.package --validate-only $GITHUB_WORKSPACE/cratedb-tableau-connector/cratedb_jdbc
+          
+          cat packaging_logs.txt
+          // If validation fails, stop the job.
+          grep 'validation failed' packaging_logs.txt && exit 1
```

Comment:
Is it possible to have a positive test when validation suceeded, and fail otherwise?

Reply:
yep, fixed now thanks!

Reply:
The code still says `grep 'validation failed'`. I was asking for inverting this, i.e. checking for a positive string label instead, and failing if it is **not contained** within the log output. Is that possible?

Reply:
Maybe checking the exit code of `python -m connector_packager.package`?

Reply:
I forgot that grep itself does exit code 0 when it finds something and 1 when it doesnt, so a `grep 'Validation succeeded' log.txt` suffices